src/Microsoft.DotNet.Build.Tasks.Feed/src/model/PublishingConstants.cs

```diff
@@ -566,6 +570,17 @@ public class PublishingConstants
                 FeedDotNetToolsSymbols,
                 FeedForChecksums,
                 FeedForInstallers),
+
+            // ".NET Libraries",
+            new TargetChannelConfig(
+                1648,
+                PublishingInfraVersion.All,
+                string.Empty,
```

Comment:
```suggestion
                akaMSChannelName: string.Empty,
```
[nit] for consistency with other channels that don't publish links?